src/queries/metApi.ts

```diff
@@ -0,0 +1,24 @@
+import { gql } from '@apollo/client';
+
+//Format: What where
+export const WEATHER_MET_API = gql`
+    query {
+        forecast {
+            forecastProperties {
+                meta {
+                    updatedAt
+                }
+                timeseries {
+                    time
+                    forecastData {
+                        instant {
+                            details {
+                                airTemperature
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+`;
```

Comment:
I'd also recommend typing the result in this file, as @blauks witnessed in ekskom.

So you export the GraphQL query as well as 

```ts
export type WeatherMetApi = {
  // type type type
}
```